src/drivers/wdm/queries/PendingStatusError/driver_snippet.c

```diff
@@ -0,0 +1,49 @@
+//
+// driver_snippet.c
+//
+
+
+//Template. Not called in this test.
+void top_level_call(){}
+
+
+//Passing Case
+_Dispatch_type_(IRP_MJ_SYSTEM_WRITE)
+DRIVER_DISPATCH DispatchWrite;
+NTSTATUS
+DispatchWrite (
+    _In_ PDEVICE_OBJECT DeviceObject,
+    _Inout_ PIRP Irp
+    )
+{  
+
+    UNREFERENCED_PARAMETER(DeviceObject);
+    PAGED_CODE();
+    //The call below represents a passing case for PendingStatusError.
+    IoMarkIrpPending(Irp);
+
+    //...

```

Comment:
What does this indicate?

Reply:
The c28143 CA rule checks if routines that call IoMarkIrpPending are returning STATUS_PENDING at some point/path after the call. I added that call to check that.

Reply:
I mean the "//..."
If it is not needed, please remove.

Reply:
Oh that was to imply other code would go here... but I just removed it.